### PR TITLE
start scrubbing use case language

### PR DESCRIPTION
**Pull request by slack**
First pass at scrubbing the use cases to match the glossary... Probably best to view this in 'word diff' mode: https://github.com/kubernetes-incubator/service-catalog/pull/26/files?w=1

### CODE REVIEW

File: docs/v1/use-cases.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 * TODO: scrub doc post glossary
 * TODO: add CF SB API as the lingua franca for blackbox services
 
+## Personas
+
+* User
+* Developer
```
Comment (duglin) [inline docs/v1/use-cases.md]: I suspect that just "Application Manager" (or Operator perhaps) might be sufficient to cover both of these.

File: docs/v1/use-cases.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
+* Broker Operator
+* Service Provider
```
Comment (duglin) [inline docs/v1/use-cases.md]: For blackbox I would suggest that we don't draw a distinction between the person who manages the service broker and the person who owns the backend service - all we see is the service broker. Whether we call that "Broker Operator" or "Service Provider"....either works but Service Provider is the term I'm more used to.

File: docs/v1/use-cases.md (hunk 1 above)
Comment (slack) [inline docs/v1/use-cases.md]: Yeah, I think it makes sense to use "Application operator" here.

File: docs/v1/use-cases.md (hunk 2 above)
Comment (duglin) [inline docs/v1/use-cases.md]: LOL ok - one less Service ...   would be good

Comment (pmorie) [inline docs/v1/use-cases.md]: One less usage of service is grand, +1

**Comment by pmorie** [comment]
@jwforres - are we missing any personas here?

File: docs/v1/use-cases.md (hunk 2 above)
Comment (arschles) [inline docs/v1/use-cases.md]: +1 from over here too

Comment (jwforres) [inline docs/v1/use-cases.md]: I wouldn't have "User" as a persona.  What makes user different than developer, that set of things is the persona you are missing.

**Comment by slack** [comment]
Re: User vs Developer, I think they are the same. I built the list by simply pulling "As an X" references. Happy to replace User with Developer or Application Operator (from the glossary PR #25).

File: docs/v1/use-cases.md [hunk 3]
```diff
@@ -1,60 +1,69 @@
...
 
-TODO: add glossary
-TODO: scrub doc post glossary
-TODO: add CF SB API as the lingua franca for blackbox services
+* TODO: scrub doc post glossary
+* TODO: add CF SB API as the lingua franca for blackbox services
+
+## Personas
+
+* User
```
Comment (pmorie) [inline docs/v1/use-cases.md]: I think @jwforres is right - let's scrub user and dissolve the 'user' use cases in one of other the existing personas or a new one.

**Comment by arschles** [comment]
LGTM

**Comment by slack** [comment]
Ok, pushed persona updates. PTAL and I'll squash/merge.

**Comment by duglin** [comment]
LGTM

**Pull request merged**